xml/security-barbican.xml

```diff
@@ -484,22 +545,22 @@ ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
       </entry>
       <entry>
        <para>
-        Barbican specific role with privilege to create/modify/list keys,
-        certificates.
+        &secret_store; specific role with privilege to
+        create/modify/list/delete keys, certificates.
```

Comment:
please replace by `create, modify, list, and delete` -- the current version looks weird and will lead to odd line breaks. (again, not your fault)